salvo/src/lib/run_benchmark.py

```diff
@@ -9,7 +9,9 @@
 
 from src.lib.benchmark import fully_dockerized_benchmark as fulldocker
 from src.lib.benchmark import scavenging_benchmark as scavenging
+from src.lib.benchmark import binary_benchmark as binbench
```

Comment:
Sorry I should have noted this in the previous review round. We also shouldn't rename imports, the style guide only calls out two reasons for renaming - when they are inconveniently long and when there are two imports with the same name. These don't seem to meet either of the exceptions. Keeping import names helps the reader to quickly find the module without the indirection.

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#224-decision

Reply:
Would you like me to change all of the benchmark import names while we're here? Or is that out of scope for this PR?

Reply:
It might be best not to extend the scope of this PR. We can either just change the one we are adding, or we can file an issue to change all of them later.

Reply:
> It might be best not to extend the scope of this PR. 

Agreed. It's nice to have fairly atomic commits in the history, IMO.
Since I'm already going to follow up with that doc fix, I'll change it for the binary benchmark, and I can create an issue for the rest of the reformatting in `run_benchmark.py`

Reply:
Issue created: https://github.com/envoyproxy/envoy-perf/issues/114